relay-event-normalization/src/event.rs

```diff
@@ -200,15 +200,17 @@ fn normalize(event: &mut Event, meta: &mut Meta, config: &NormalizationConfig) -
     transactions_processor.process_event(event, meta, ProcessingState::root())?;
 
     // Check for required and non-empty values
-    schema::SchemaProcessor.process_event(event, meta, ProcessingState::root())?;
+    schema::SchemaProcessor
+        .process_event(event, meta, ProcessingState::root())
+        .ok();
```

Comment:
Can we not make the processor not return a `Result` here, just because the processor currently never returns an error, doesn't mean it will forever and we have no guarantee for this.

Reply:
As discussed offline (posting here for future reference): this is a great idea but requires a bigger refactor, and that is too much work to get into at this point.